doc/manual/Makefile

```diff
@@ -15,22 +15,27 @@ docs: config
 # KLUDGE: call make instead of a dependency so that the html target
 # will run after config from inside the doc target.
 docs distdocs:
-	$(MAKE) html
+	$(MAKE) html team
```

Comment:
Having `html` refer to the admin/jury manual and `team` to the (PDF) team manual is a bit weird. Maybe rename `html` to `admin` or so?

Reply:
So the sources of the team manual are now part of the entire manual. So html refers to the HTML version of the built manual. The team target builds a PDF of specifically the team manual.